apps/python/bin/chat.py

```
@@ -52,6 +52,28 @@ def build_cmd(target_date: str, briefing_file: Path, session_file: Path) -> list
     ]
 
 
+def run_claude(target_date: str, briefing_file: Path, session_file: Path) -> int:
+    cmd = build_cmd(target_date, briefing_file, session_file)
+    if "--resume" not in cmd:
+        return subprocess.run(cmd).returncode
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)